ansible/site.yml

```diff
@@ -0,0 +1,82 @@
+---
+- hosts: primary
+  gather_facts: no
+
+  tasks:
+  - name: install python 2
+    raw: test -e /usr/bin/python || (apt -y update && apt install -y python2.7)
+    become: yes
+
+- name: Provision ISIC Archive
+  hosts: primary
+
+  vars:
+    mongo_data_path: "/srv/mongodb"
+    bind_node_modules: false
+    ansible_python_interpreter: /usr/bin/python2.7
+
+  pre_tasks:
+    - name: Find path to storage volume
+      shell: |
+        readlink -f $(ls /dev/disk/by-id/nvme-Amazon_Elastic_Block_Store_{{ storage_volume_id }})
+      changed_when: false
+      become: yes
+      register: volume_path_cmd
+
+    - set_fact:
+        volume_path: "{{ volume_path_cmd.stdout }}"
+
+    - name: Mount storage filesystem
+      mount:
+        path: /srv
+        src: "{{ volume_path }}"
+        fstype: ext4
+        state: mounted
+      notify: Restart Mongo
+      become: yes
+      become_user: root
+
+  roles:
+    - role: isic
+      isic_archive_path: "{{ ansible_user_dir }}/isic_archive"
+      isic_archive_assetstore_path: "/srv/girder/default"
+      isic_cors_allow_origin: "https://dermannotator.org, https://www.isic-archive.com, http://display.isic-archive.com"
+      isic_cors_allow_methods: "HEAD, GET, POST, PUT, DELETE"
+
+    - role: certbot
+      domain: "{{ domain_name }}"
+      email: brian.helba@kitware.com
+
+    - role: nginx
+      site_hostname: "{{ domain_name }}"
+      upstream_proxy: true
+      redirect_canonical: true
+      ssl: true
+
+  post_tasks:
+    - name: Chown Girder assetstore dir
+      file:
+        path: /srv/girder
+        mode: 0755
+        owner: ubuntu
+        group: ubuntu
+        state: directory
+      become: yes
+      become_user: root
+
+    - name: Chown mongodb data dir
+      file:
+        path: "{{ mongo_data_path }}"
+        mode: 0700
+        owner: mongodb
+        group: mongodb
+        state: directory
+        recurse: yes
+      become: yes
+      become_user: root
+
+    - name: Activate virtualenv on login
```

Comment:
In production, this shouldn't actually need to be set, since we *shouldn't* be SSHing manually at all. However, until we're in that perfect world, I'm happy to leave this as a convenience.